framework/db/Exception.php

```diff
@@ -27,9 +27,9 @@ class Exception extends \yii\base\Exception
      * @param string $message PDO error message
      * @param array $errorInfo PDO error info
      * @param string $code PDO error code
-     * @param \Exception $previous The previous exception used for the exception chaining.
+     * @param \Throwable|\Exception $previous The previous exception used for the exception chaining.
      */
-    public function __construct($message, $errorInfo = [], $code = '', \Exception $previous = null)
+    public function __construct($message, $errorInfo = [], $code = '', $previous = null)
```

Comment:
why is this needed?

Reply:
Because PHP 8 introduced \Error and other native error types. These aren't extended from \Exception but passed here.